app/[locale]/sign-in/page.tsx

```diff
@@ -65,7 +65,15 @@ export default function AuthenticationPage() {
             i18n("MustContainLeast"),
             "Password",
             i18nCommon("One"),
-            i18nCommon("UppercaseLetter")
+            i18nCommon("UppercaseChar")
+          ),
+        })
+        .refine((password) => /[a-z]/.test(password), {
+          message: sprintf(
+            i18n("MustContainLeast"),
+            "Password",
+            i18nCommon("One"),
+            i18nCommon("LowerCaseChar")
```

Comment:
`UppercaseChar` and `LowerCaseChar` are not properly localized
![image](https://github.com/ferrariofilippo/woel/assets/102259289/8c789d43-dc29-4bff-9a1c-e76dc3615067)

Edit. Can you fix also the length constraint (it says `one character` instead of `eight characters`